presto-main/src/main/java/com/facebook/presto/execution/NodeScheduler.java

```diff
@@ -195,12 +182,11 @@ public boolean apply(Node node)
         }
 
         /**
-         * Identifies the nodes for runnning the specified splits.
+         * Identifies the nodes for running the specified splits.
          *
          * @param splits the splits that need to be assigned to nodes
-         *
          * @return a multimap from node to splits only for splits for which we could identify a node to schedule on.
-         *      If we cannot find an assignment for a split, it is not included in the map.
+         * If we cannot find an assignment for a split, it is not included in the map.
```

Comment:
I think this should be aligned to the `@return`. That's how it is shown in the "How to Write Doc Comments for the Javadoc Tool". Though it might be better to move this sentence to the description (and remove the period for `@return`).